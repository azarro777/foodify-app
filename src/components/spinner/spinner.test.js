import React from 'react';
import Spinner from './spinner';
import { shallow } from 'enzyme';

describe('Testing <Spinner/> component', () => {
  it('Spinner have rendered correctly', () => {
    const spinner = shallow(<Spinner />);
    expect(spinner).toMatchSnapshot();
  });
});
