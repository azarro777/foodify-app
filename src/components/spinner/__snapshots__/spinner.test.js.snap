// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Spinner/> component Spinner have rendered correctly 1`] = `ShallowWrapper {}`;
